### DISCUSSION TITLE
Setting gas price to zero for local network fork

### DISCUSSION THREAD

**Discussion by scheuclu** [foundry-rs/foundry#5526] [Help]
I am trying to switch from hardhat to anvil.

We write a simulation software that forks the chain at a particular time and then interacts with DeFi pools to simulation trading behaviours.

One thing that was very helpful in hardhat was setting the gasprice to 0.

Despite my best efforts, I have not yet been able to launch a network with anvil that has this property.

Any help/advice would be greatly appreciated.

**Comment by maa105** [comment]
This is indeed very useful. I have just had a usecase where this is needed. Any updates on this where you able to figure it out?
